index.html

```diff
@@ -0,0 +1,356 @@
+            <div class="videos">
+                <div class="video ">
+                    <a href="PlayPage.html">
+                    <div class="thumbnail">
+                        <img src="http://img.youtube.com/vi/ObAcg1k25YI/maxresdefault.jpg" alt="" />
+                    </div>
+                    <div class="title">
+                        <h3>
+                            Home — Another Kid | Free Background Music | Audio Library Release
+                        </h3>
+                        <a href="">
+                            Audio Library Plus — Official Releases
+
+                        </a>
+                        <span> 32K Views • 2 weeks ago </span>
+                    </div>
+                    </a>
+                </div>
+                <div class="video ">
+                    <a href="PlayPage.html">
+                    <div class="thumbnail">
+                        <img src="http://img.youtube.com/vi/ObAcg1k25YI/maxresdefault.jpg" alt="" />
+                    </div>
+                    <div class="title">
+                        <h3>
+                            Home — Another Kid | Free Background Music | Audio Library Release
+                        </h3>
+                        <a href="">
+                            Audio Library Plus — Official Releases
+
+                        </a>
+                        <span> 32K Views • 2 weeks ago </span>
+                    </div>
+                    </a>
+                </div>
+                <div class="video ">
+                    <a href="PlayPage.html">
+                    <div class="thumbnail">
+                        <img src="http://img.youtube.com/vi/ObAcg1k25YI/maxresdefault.jpg" alt="" />
+                    </div>
+                    <div class="title">
+                        <h3>
+                            Home — Another Kid | Free Background Music | Audio Library Release
+                        </h3>
+                        <a href="">
+                            Audio Library Plus — Official Releases
+
+                        </a>
+                        <span> 32K Views • 2 weeks ago </span>
+                    </div>
+                    </a>
+                </div>
+                <!--div class="video ">
+                    <a href="PlayPage.html">
+                    <div class="thumbnail">
+                        <img src="http://img.youtube.com/vi/ObAcg1k25YI/maxresdefault.jpg" alt="" />
+                    </div>
+                    <div class="title">
+                        <h3>
+                            Home — Another Kid | Free Background Music | Audio Library Release
+                        </h3>
+                        <a href="">
+                            Audio Library Plus — Official Releases
+
+                        </a>
+                        <span> 32K Views • 2 weeks ago </span>
+                    </div>
+                    </a>
+                </div-->
```

Comment:
I advise you not to leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.